docs/book/src/getting-started.md

```diff
@@ -0,0 +1,537 @@
+# Getting Started 
+
+## Overview
+
+By following the [Operator Pattern][k8s-operator-pattern], it’s possible not only to provide all expected resources but also to manage them dynamically, programmatically, and at execution time. To illustrate this idea, imagine if someone accidentally changed a configuration or removed a resource by mistake; in this case, the operator could fix it without any human intervention. Before following the next steps ensure that you have the Operator-SDK tool with a version >= 1.0 installed locally. [To install, check its guide](#installation-guide).
+
+## Sample Project
+
+We will create a sample project to let you know how it works. This sample will:
+
+- Reconcile a Memcached CR
+- Create a Deployment with the Memcached image
+- Not allow more instances than the size defined in the CR which will be applied
+- Update the Memcached CR status 
+
+Following the steps.
```

Comment:
```suggestion


###Following the steps.
```

in preview following the steps feels like out of plsce it should br near to create a project.

Reply:
The text following the steps should not be a subsection. It is just a description to let the audience be aware that the steps will be provided bellow.

Reply:
![image](https://github.com/kubernetes-sigs/kubebuilder/assets/109643863/80d9fce8-de20-4318-b20b-55d9174bd611)

To me this doesnt looks good. I mean the format. I think we should either add something after following the steps or remove it or make it near create the project. What say?